### DISCUSSION TITLE
v1.15.3

### DISCUSSION THREAD

**Discussion by Victor-IX** [Victor-IX/Blender-Launcher-V2#2] [Announcements]
# What's Changed

## ⚠Info:
- Some Blender Websites are still not fully working after the recent DDoS attack you may not be able to download some of the Blender builds
- It can take some time to get all the Blenders versions so be patient when the program is checking for new versions
- Only the Windows version is available for now, I need to set up a Linux environment to build the Linux version
- Expect bugs, I was not able to test everything
- **Blender Foundation is currently blocking the request on the /release website for the stables versions making it impossible to retrieve those versions, I'm currently trying to reach out to Blender to check with them how we can fix this issue**

## ⚙️Other:
- Fore Disable Check For New Build Automatically (even if you reenable this option it will not work), this has been made to lower the number of requests

<hr /><em>This discussion was created from the release <a href='https://github.com/Victor-IX/Blender-Launcher/releases/tag/v1.15.3'>v1.15.3</a>.</em>

**Comment by virtualrepublic** [comment]
Thanks so much for taking over this useful tool and fixing the issues. Appreciated! Cheers from Germany!
Michael

**Comment by eisenbruch** [comment]
Thanks for your work! Hoping that it becomes fully working again. I can't live without my blender launcher :)

**Comment by eisenbruch** [comment]
Loving the progress that I have seen over the past 2 days in pull requests, etc.

Any idea when we'll get the next release?

**Reply by Victor-IX** [reply]
I will make a new one today with Linux and Windows build
